data-access/seedwork/services-seedwork-datastore-mongodb/interfaces/base.ts

```diff
@@ -44,3 +44,8 @@ export const Patterns = {
   EMAIL_PATTERN: /^[a-zA-Z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?(?:\.[a-zA-Z0-9](?:[a-zA-Z0-9-]{0,61}[a-zA-Z0-9])?)*$/,
   GUID_PATTERN: /^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/i,
 };
+
+
+export type FindQuery = {
```

Comment:
Let's discuss this when Nick is back, I believe, we have existing means of finding in the list of IDs